django_fido/views.py

```diff
@@ -99,7 +108,11 @@ def get(self, request: HttpRequest) -> JsonResponse:
         try:
             request_data, state = self.create_fido2_request()
         except ValueError as error:
-            return JsonResponse({'error': force_text(error)}, status=BAD_REQUEST)
+            return JsonResponse({
+                'error_code': getattr(error, 'error_code', None),
+                'message': force_text(error),
+                'error': force_text(error),
```

Comment:
I would at least add a note here that the `error` is deprecated.

Also a note in CHANGELOG should be added.

Reply:
Fixed.